packages/dev/core/src/Shaders/ShadersInclude/pbrDirectLightingFalloffFunctions.fx

```diff
@@ -46,6 +46,13 @@ float computeDirectionalLightFalloff_Standard(vec3 lightDirection, vec3 directio
     return falloff;
 }
 
+float computeDirectionalLightFalloff_IES(vec3 lightDirection, vec3 directionToLightCenterW, sampler2D iesLightSampler)
+{
+    float cosAngle = maxEps(dot(-lightDirection, directionToLightCenterW));
+	float angle = acos(cosAngle) / PI;

```

Comment:
Same comment than above.

Reply:
nope the data is from 0 to 180 deg so we want from 0 to 0.5